modules/dynamic-plugins/con-catalog-searching-and-filtering.adoc

```diff
@@ -0,0 +1,12 @@
+= Searching and filtering the plugins
+
+== Searching by plugin name
```

Comment:
If the title starts with a gerund, it should typically be a procedural module. Instead, we could use the following suggestion. You might consider modularizing this section.
```suggestion
== Search by plugin name
```